lib/src/client.dart

```diff
@@ -30,6 +30,13 @@ abstract class Client {
   /// [BrowserClient] if `dart:html` is available.
   factory Client() => new IOClient();
 
+  /// Creates a new client handler.
```

Comment:
Maybe "Creates a new `Client` from a [handler] callback." The client itself isn't a handler in the type-system sense, so we should avoid implying that in the docs.